coral/src/app/layout/header/ProfileDropdown.test.tsx

```diff
@@ -13,11 +13,15 @@ jest.mock("src/domain/auth-user/auth-user-api");
 
 const mockLogoutUser = logoutUser as jest.MockedFunction<typeof logoutUser>;
 
-const mockedUseToast = jest.fn();
+const mockedUseToastContext = jest.fn();
 jest.mock("@aivenio/aquarium", () => ({
   ...jest.requireActual("@aivenio/aquarium"),
-  useToast: () => mockedUseToast,
+  useToastContext: () => ({
+    toast: mockedUseToastContext,
+    onDismiss: jest.fn(),
```

Comment:
It would be awesome if you could also test the correct call to dismiss for the toast. So this would need to be a mock, something like `const mockDismiss = jest.fn()` 

and in the test case for the correct error notification we can check that the dismiss is called like expected:

```
it("shows error notification to the user", async () => {
(...)
(...)
(...)
    expect(mockDismiss).toHaveBeenCalledWith("logout")   
    expect(mockToast).toHaveBeenCalledWith(
(...)
```

Reply:
done!